types/index.d.ts

```diff
@@ -411,12 +411,12 @@ declare module 'react-base-table' {
      * A callback function when resizing the column width
      * The handler is of the shape of `({ column, width }) => *`
      */
-    onColumnResize?: (args: { column: ColumnShape<T>; width: number }) => void;
+    onColumnResize?: (args: { column: ColumnShape<T>; width: number, headerIndex: number }) => void;
```

Comment:
@MostFrumiousBandersnatch Do you think it makes sense to make this argument optional or at least have a default value?

Reply:
@sklinov I consider this value as [always available](https://github.com/datafold/react-base-table/blob/master/src/TableHeader.js#L19-L25)